Jetcaster/wear/src/main/java/com/example/jetcaster/ui/library/LibraryScreen.kt

```diff
@@ -0,0 +1,198 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetcaster.ui.library
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.MusicNote
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.wear.compose.foundation.lazy.items
+import androidx.wear.compose.material.ChipDefaults
+import androidx.wear.compose.material.MaterialTheme
+import androidx.wear.compose.material.Text
+import com.example.jetcaster.R
+import com.example.jetcaster.core.model.PodcastInfo
+import com.google.android.horologist.composables.PlaceholderChip
+import com.google.android.horologist.compose.layout.ScalingLazyColumn
+import com.google.android.horologist.compose.layout.ScalingLazyColumnDefaults
+import com.google.android.horologist.compose.layout.ScalingLazyColumnDefaults.listTextPadding
+import com.google.android.horologist.compose.layout.ScalingLazyColumnState
+import com.google.android.horologist.compose.layout.ScreenScaffold
+import com.google.android.horologist.compose.layout.rememberResponsiveColumnState
+import com.google.android.horologist.compose.material.Chip
+import com.google.android.horologist.compose.material.ListHeaderDefaults
+import com.google.android.horologist.compose.material.ResponsiveListHeader
+import com.google.android.horologist.images.base.paintable.DrawableResPaintable
+import com.google.android.horologist.images.base.util.rememberVectorPainter
+import com.google.android.horologist.images.coil.CoilPaintable
+import com.google.android.horologist.media.ui.screens.entity.EntityScreen
+
+@Composable
+fun LibraryScreen(
+    onLatestEpisodeClick: () -> Unit,
+    onYourPodcastClick: () -> Unit,
+    onUpNextClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    libraryScreenViewModel: LibraryViewModel = hiltViewModel()
+) {
+    val uiState by libraryScreenViewModel.uiState.collectAsState()
+
+    val columnState = rememberResponsiveColumnState(
+        contentPadding = ScalingLazyColumnDefaults.padding(
+            first = ScalingLazyColumnDefaults.ItemType.Text,
+            last = ScalingLazyColumnDefaults.ItemType.Chip,
+        ),
+    )
+
+    when (val s = uiState) {
+        is LibraryScreenUiState.Loading -> LoadingScreen(columnState = columnState)
+        is LibraryScreenUiState.NoSubscribedPodcast ->
+            ScreenScaffold(scrollState = columnState, modifier = modifier) {
```

Comment:
Nit: This can be set as it's own separate composable (e.g. `NoSubscribedPodcast`) to improve readability. Same comment for the `Ready` state below.